source/_patterns/04-pages/article--magazine.json

```diff
@@ -90,10 +90,21 @@
         {
           "text": "Submit my research",
           "path": "#",
-          "classes": "nav-secondary__item nav-secondary__item--button",
+          "classes": "nav-secondary__item",
```

Comment:
Do we need `classes` as a property anymore?

Reply:
In some cases, yes (see https://github.com/elifesciences/pattern-library/blob/develop/source/_patterns/04-pages/article--magazine.json#L45). The more general case (primary/secondary) might be simplified through refactoring, but I think that's a separate task.